docs/deployment.md

```diff
@@ -41,10 +46,13 @@ The main changes are:
 This inventory will attempt to automatically route all SSH access via `idr-proxy`, using metadata attached to the servers in the [IDR openstack-idr-instance](https://github.com/IDR/ansible-role-openstack-idr-instance) role.
 
 ### Static inventory
-This is recommended if you have provisioned the servers yourself, or if you are unfamiliar with setting up an Ansible dynamic inventory.
+Use this if you have provisioned the servers yourself, or if you are unfamiliar with setting up an Ansible dynamic inventory.
 Modify the `Hosts` section in the [example static inventory](../inventories/ansible-hosts).
 You should not need to change the `Host groups` section.
 
+Note we do not currently test this as we always use Ansible for provisioning resources.
+
+
 ## Ansible deployment example
```

Comment:
Eventually, once naming is decided, this should likely be "Core deployment" (or similar)